scripts/firebase-admin/updateGeneralCourt.ts

```diff
@@ -0,0 +1,92 @@
+import { Script } from "./types"
+import { currentGeneralCourt } from "../../functions/src/shared"
+
+export const script: Script = async ({ db, args }) => {
+  console.log(`Update General Court ${currentGeneralCourt} in progress...`)
+  try {
+    const generalCourts = await db
+      .collection(`/generalCourts/${currentGeneralCourt}/members`)
+      .get()
+    console.log(`${generalCourts.size} members fetched`)
+
+    // check if generalCourts is valid
+    if (generalCourts.empty) {
+      console.log("ERROR: No members found in the collection.")
+      return
+    }
+
+    const courtMembersMap = new Map<string, any>()
+    for (const member of generalCourts.docs) {
+      const data = member.data()
+      // store the member data in a map with the district as the key
+      courtMembersMap.set(data.content["District"], data)
+    }
+
+    // prepare update
+    const writer = db.bulkWriter()
+
+    console.log("Fetching all profiles ...")
+    const users = await db.collection("profiles").get()
+    console.log(`${users.size} profiles fetched`)
+
+    for (const user of users.docs) {
+      let data = user.data()
+      console.log("User info:", user.id)
+
+      if (data.hasOwnProperty("senator")) {
+        const senator = data.senator
+        if (courtMembersMap.has(senator.district)) {
```

Comment:
I don't feel that this is precisely the right comparison - we only want to update a Profile if the profile has an existing senator and/or representative who DOESN'T exist in the new general court. 

We expect the district to be the same for both the court 193 members and court 194 members, so we should likely be checking if the name in the `courtMembersMap` for the profile's districts matches the name on the profile's rep - if they match, we don't need to update this member on this profile.

Reply:
As a side-note, it'd be nice to track the counts of users we're updating/not updating to log out at the end (e.g. something like `numUsersWithoutSenator`/`numUsersWithUpToDateSenator`/`numUsersUpdated`) - that would be useful for the dry run to get a sense of how many updates to expect.